app/src/main/java/com/framgia/fsalon/utils/Utils.java

```diff
@@ -45,4 +55,27 @@ public static void disableShiftMode(BottomNavigationView view) {
             Log.e("BNVHelper", "Unable to change value of shift mode", e);
         }
     }
+
+    public static int createTimeStamp(@SchedulerViewModel.TabFilter int tab) {
+        Calendar calendar = Calendar.getInstance();
+        switch (tab) {
```

Comment:
WARNING: [SwitchIntDef] Switch statement on an `int` with known associated constant missing case `TabFilter.TAB_CALENDAR`
 This check warns if a `switch` statement does not explicitly include all the values declared by the typedef `@IntDef` declaration.